src/applications/static-pages/facilities/tests/NearbyLocations.unit.spec.jsx

```diff
@@ -0,0 +1,528 @@
+            mainFacilityApiId="vba_325"
+            mainPhone="800-827-1000"
+          />
+        </Provider>,
+      );
+
+      // wait for useEffect
+      setTimeout(() => {
+        wrapper.update();
+        expect(wrapper.find('h2').text()).to.equal('Other nearby VA locations');
+        expect(
+          wrapper
+            .find('h3')
+            .first() // it does 3 requests and gets back 1 facility (that is the mock) for Health, VetCenter, and Cemetery
+            .text(),
+        ).to.equal('Cleveland VA Clinic-Euclid');
+        wrapper.unmount();
+        done();
+      }, 100);
+    });
+  });
+});
+describe('VAFacilityInfoSection', () => {
+  it('should render VAFacilityInfoSection', () => {
+    const mainPhone = '800-827-1000';
+    const vaFacility = buildFacility(fetchedVALocationsData[0], 3);
+    const wrapper = mount(
+      <VAFacilityInfoSection mainPhone={mainPhone} vaFacility={vaFacility} />,
+    );
+    expect(wrapper.find('h3').exists()).to.be.true;
+    expect(wrapper.find('strong').exists()).to.be.true;
+    const telephone = wrapper.find('va-telephone');
+    const telephoneProps = telephone.props();
+    expect(telephoneProps).to.deep.equal({
+      contact: '216-391-0264',
+      extension: '',
+    });
+
+    wrapper.unmount();
+  });
+});
+describe('VAFacilityPhone', () => {
+  it('should render VAFacilityPhone with a phone number title', () => {
+    const wrapper = mount(
+      <VAFacilityPhone phoneNumber="301-000-0000" phoneTitle="Main phone" />,
+    );
+    expect(wrapper.find('strong').exists()).to.be.true;
+    const telephone = wrapper.find('va-telephone');
+    const telephoneProps = telephone.props();
+    expect(telephoneProps).to.deep.equal({
+      contact: '301-000-0000',
+      extension: '',
+    });
+    wrapper.unmount();
+  });
+  it('should render VAFacilityPhone with NO phone number title', () => {
+    const wrapper = mount(<VAFacilityPhone phoneNumber="301-000-0000" />);
+    expect(wrapper.find('strong').exists()).to.be.false;
+    const telephone = wrapper.find('va-telephone');
+    const telephoneProps = telephone.props();
+    expect(telephoneProps).to.deep.equal({
+      contact: '301-000-0000',
+      extension: '',
+    });
+    wrapper.unmount();
+  });
+});
+describe('processPhoneNumber', () => {
+  it('should process phone number strings into phone and extension', () => {
+    const phoneNumber = '800-827-1000';
+    const phoneNumberNoDashes = '8008271000';
+    const extension = '123';
+    const temp1 = phoneNumber;
+    const temp2 = `${phoneNumber}x${extension}`;
+    const temp3 = `${phoneNumber} ext ${extension}`;
+    const temp4 = `${phoneNumber} ext. ${extension}`;
+    const temp5 = `${phoneNumber} extension ${extension}`;
+    const temp6 = `${phoneNumber} x. ${extension}`;
+    const temp7 = `${phoneNumber}, x${extension}`;
+    const temp8 = `${phoneNumber}, ext${extension}`;
+    const temp9 = `${phoneNumber}, ext. ${extension}`;
```

Comment:
**Suggestion:** Putting all these `temp` strings in an array, and then looping through the array to render all the different tests could help cut down on how long this test is.

Reply:
Sure, can do that.